src/appkit.cs

```diff
@@ -12913,6 +13297,39 @@ interface NSSliderCell {
 		[Export ("barRectFlipped:")]
 		CGRect BarRectFlipped (bool flipped);
 	}
+
+	[Mac (10,12,1)]
+	[BaseType (typeof(NSTouchBarItem))]
+	interface NSSliderTouchBarItem
+	{
+		[Export ("initWithIdentifier:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (string identifier);
+
+		[Export ("slider", ArgumentSemantic.Strong)]
+		NSSlider Slider { get; set; }
+
+		[NullAllowed, Export ("label")]
+		string Label { get; set; }
+
+		[NullAllowed, Export ("minimumValueAccessory", ArgumentSemantic.Strong)]
+		NSSliderAccessory MinimumValueAccessory { get; set; }
+
+		[NullAllowed, Export ("maximumValueAccessory", ArgumentSemantic.Strong)]
+		NSSliderAccessory MaximumValueAccessory { get; set; }
+
+		[Export ("valueAccessoryWidth")]
+		double ValueAccessoryWidth { get; set; }
+
+		[NullAllowed, Export ("target", ArgumentSemantic.Weak)]
+		NSObject Target { get; set; }
```

Comment:
So in many other places in appkit.cs we expose these (NSControl among other places), so how about keeping them public but making nicer overrides.